docker-compose.env.example

```diff
@@ -0,0 +1,38 @@
+## database related variables ##
+
+# variables prefixed with MYSQL_ are used by both database and web containers
+# variables prefixed with DATABASE_ are used exlusively by the web container
+
+MYSQL_DATABASE=osem
+MYSQL_USER=osem
+MYSQL_PASSWORD=changemeimmediately
+MYSQL_ROOT_PASSWORD=changemeevenmoreimmediately
+
+# the following settings should not be modified unless the database service
+# is renamed in docker-compose.yml or you plan to use an external database
+DATABASE_HOST=database
+DATABASE_PORT=3306
+
+
+## OSEM options ##
+# you can configure any option described in this document here instead of
+# having to create a .env file:
+# https://github.com/openSUSE/osem/blob/master/dotenv.example
+
+OSEM_NAME=Dockerized OSEM
+OSEM_HOSTNAME=http://localhost:9292
+OSEM_ERRBIT_HOST=localhost
+SECRET_KEY_BASE=changemechangemechangeme
+
+# these settings work for the MailHog server that is enabled by default in
+# docker-compose.yml
+# if you do not plan to use MailHog (you most likely don't want to), you need
+# to change these settings to use an external working mailserver, otherwise
+# your users are going to see the HTTP status 500 page
+# you should comment out or remove the mailhog service from docker-compose.yml,
+# too
+OSEM_EMAIL_ADDRESS=osem@mailhog
+OSEM_SMTP_ADDRESS=mailhog
+OSEM_SMTP_PORT=1025
+OSEM_SMTP_USERNAME=mailhog
+OSEM_SMTP_PASSWORD=mailhog
```

Comment:
Well, not necessarily. The mailhog test system provided by the default configuration for example works without it. I am not necessarily sure whether it picks the right authentication method automatically, or its default value works. I'd suggest to add it as a commented line, as it is an optional value.